website/blog/2020-10/00/_index.md

```diff
@@ -0,0 +1,111 @@
+- Validation Provider Webhooks
+- Project Namespace
+
+**Seed Cluster**
+- Gardenlet
+- Seed API server 
+  - every Shoot Control Plane has a dedicated namespace in the Seed.
+- Cloud Provider (owned by Stakeholder). 
+  - Arguably part of the Shoot cluster but used by components in the Seed cluster to create the infrastructure for the Shoot.
+- [Gardener DNS extension](https://github.com/gardener/external-dns-management)
+- Provider Extension (such as [gardener-extension-provider-aws](https://github.com/gardener/gardener-extension-provider-aws))
+- [Gardener Extension ETCD Druid](https://github.com/gardener/etcd-druid)
+- [Gardener Resource Manager](https://github.com/gardener/gardener-resource-manager)
+- Operating System Extension (such as [gardener-extension-os-gardenlinux](https://github.com/gardener/gardener-extension-os-gardenlinux))
+- Networking extension (such as [gardener-extension-networking-cilium](https://github.com/gardener/gardener-extension-networking-cilium))
+- [Machine Controller Manager](https://github.com/gardener/machine-controller-manager)
+- ContainerRuntime Extension (such as [gardener-extension-runtime-gvisor](https://github.com/gardener/gardener-extension-runtime-gvisor))
+- Shoot API server (in the Shoot Namespace in the Seed cluster)
+
+**Shoot Cluster**
+- Cloud Provider compute API (owned by Stakeholder) - for VM/Node creation
+- VM / Bare metal node hosted by Cloud Provider (in Stakeholder owned account)
+
+
+### How to use the diagram
+The diagram 
+- should be read from top to bottom - starting in the top left corner with the creation of the Shoot resource via the Gardener Extension API server.
+- should not require an encompassing documentation / description.
+  More detailed documentation on the components itself, can usually be found in the respective repository.
+- does not show which activities execute in parallel (many) and also does not describe the exact dependencies between the steps. 
+  This can be found out by [looking at the source code](https://github.com/gardener/gardener/blob/master/pkg/gardenlet/controller/shoot/shoot_control_reconcile.go).
+  It however tries to put the activities in a logical order of executing during the reconciliation flow. 
+
+Occasionally, there is an info box with additional information next to parts in the diagram that in my point of view require further explanation.
+Large example resource for the Gardener CRDs (e.g Worker CRD, Infrastructure CRD) are placed on the left side and are referenced by a dotted line (-----).
+
+Be aware, that Gardener is an evolving project, so the diagram will most likely be already outdated by the time you are reading this.
+Nevertheless, it should give a solid starting point for further explorations into the details of Gardener.
+
+### Flow diagram
+
+You can find the diagram [on Github.com](https://github.com/danielfoehrKn/diagrams/tree/master/gardener/shoot-reconciliation).
+There are multiple formats available.
```

Comment:
It seems worthy to place an image already here and let people take a look immediately. You mention that there are other formats so they can jump to that later if necessary. but probably 90% will be more comfortable to open a png right here.

```suggestion
You can find the diagram [on Github.com](https://github.com/danielfoehrKn/diagrams/tree/master/gardener/shoot-reconciliation). There are multiple formats available.

![gardener reconcile diagram](https://raw.githubusercontent.com/danielfoehrKn/diagrams/master/gardener/shoot-reconciliation/gardener_reconcile_with_grid.png)
```

Reply:
or rather change the markdown tag ![]() to <img> and keep the image small with style (400 width may be a good anchor). it needs to be a thumbnail hinting to open up for details not a regular picture.

Reply:
I tried that, but I cannot find a way to insert the diagram there that is readable, providing value, at all. Probably too large for any mobile or laptop device to "just view". 
Plus the picture will be outdated with the next commit to the repository. 

So not sure.

Reply:
I am very much open to it if you got a setting that I could try out.

Reply:
No the point is not to make it readable on the page, but visually hint that there's a (large) diagram if you click this thumbnail image, but you can do that here and not need to jump on another location. The point is, 100% of the people will look into the PNG first when they land on the other location anyway and quite likely it will be ok for most of them (you can zoom in/out and scroll the browser window). So UX advises that we spare them this additional click and look up for a format.

Maybe try this:
<img style="width:300px; height: auto; margin: 0;auto" src="https://raw.githubusercontent.com/danielfoehrKn/diagrams/master/gardener/shoot-reconciliation/gardener_reconcile_with_grid.png" target="_blank">Some text</a>

Play with the width and remove the margin style (for centering) if necessary. If you are still convinced it's not ok, leave it be. That's just an improvement suggestion.

Reply:
thanks added it!